tests/test_live_validation.c

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Ah good point, will adopt. Btw. there seems to be some inconsistency between LICENSE (MIT) and file headers (referring to (L)GPL? We should fix that, too - but in a separate PR